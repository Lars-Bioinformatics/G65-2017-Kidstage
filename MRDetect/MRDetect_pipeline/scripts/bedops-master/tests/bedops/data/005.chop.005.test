chr1	10	15	id-1
chr2	10	15
chr3	10	15
chr9	10	15
chrM	10	15
chrX	10	15
chrY	10	15	id-2
