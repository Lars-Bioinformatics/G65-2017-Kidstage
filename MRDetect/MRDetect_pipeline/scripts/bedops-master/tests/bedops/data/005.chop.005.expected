chr1	10	11
chr1	11	12
chr1	12	13
chr1	13	14
chr1	14	15
chr2	10	11
chr2	11	12
chr2	12	13
chr2	13	14
chr2	14	15
chr3	10	11
chr3	11	12
chr3	12	13
chr3	13	14
chr3	14	15
chr9	10	11
chr9	11	12
chr9	12	13
chr9	13	14
chr9	14	15
chrM	10	11
chrM	11	12
chrM	12	13
chrM	13	14
chrM	14	15
chrX	10	11
chrX	11	12
chrX	12	13
chrX	13	14
chrX	14	15
chrY	10	11
chrY	11	12
chrY	12	13
chrY	13	14
chrY	14	15
